src/Estimators/include/BipedalLocomotion/Estimators/FloatingBaseEstimator.h

```diff
@@ -0,0 +1,268 @@
+/**
+ * @file FloatingBaseEstimator.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_ESTIMATORS_FBE_H
+#define BIPEDAL_LOCOMOTION_ESTIMATORS_FBE_H
+
+#include <BipedalLocomotion/System/Advanceable.h>
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+#include <BipedalLocomotion/Estimators/FloatingBaseEstimatorParams.h>
+#include <BipedalLocomotion/Estimators/FloatingBaseEstimatorIO.h>
+
+#include <iDynTree/Model/Model.h>
+#include <iDynTree/KinDynComputations.h>
+#include <iDynTree/Model/JointState.h>
+#include <iostream>
+
+namespace BipedalLocomotion
+{
+namespace Estimators
+{
+
+/**
+*  FloatingBaseEstimator class contains the bare-bones implementation for
+*  a LeggedOdometry based or Extended Kalman Filter based floating base estimation
+*  algorithms for bipedal robots.
+*  @note it is assumed that if an IMU (primary) is specified in the inherited implementation,
+*  then this IMU is rigidly attached to the specified base link in the implementation
+*/
+class FloatingBaseEstimator : public BipedalLocomotion::System::Advanceable<FloatingBaseEstimators::Output>
+{
```

Comment:
Missing a virtual destructor.

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/79/commits/f46bd5b271b6e7ea77da83c41b90178157aac069